cub/cub/device/dispatch/kernels/transform.cuh

```diff
@@ -697,6 +703,8 @@ _CCCL_DEVICE void transform_kernel_ublkcp(
       };
 
       // Order of evaluation is left-to-right
+      _CCCL_PDL_GRID_DEPENDENCY_SYNC(); // only elected thread waits, but other threads wait on the barrier fulfilled by
+                                        // the bulk copy later
```

Comment:
Nitpick: I would prefer to put the comment on its own line instead of it wrapping like this. Applies throughout